src/utils/Renderers.tsx

```diff
@@ -9,6 +9,8 @@ import * as Types from "./Types";
 /////////////////
 // MapChart.tsx
 /////////////////
+let currentInstancesCounter = 0;
+let totalInstanceCounter = 0;
```

Comment:
We should move these two lines into `createVisibleRowCount`, they are not being used elsewhere, so moving these into the function which is using them could be a great option.